tasks/managed/update-infra-deployments/update-infra-deployments.yaml

```diff
@@ -20,22 +20,27 @@ spec:
     - name: snapshotPath
       type: string
       description: path to snapshot json file
+      type: string
```

Comment:
The rhtap-service-push-e2e-test test failure was caused by duplicate type: string declaration.
probably rebase didn't pass well.
it was a [fix](https://github.com/konflux-ci/release-service-catalog/pull/1561/files#diff-8729e8a878b9320cb1155101ef06fb5483c5809d59e082d54d708d41c6813919R19) which attempted to fix merge conflicts but accidentally left duplicate type key.
just remove duplication and test should pass